layers/ubuntu1604/bazel/deps_spec.yaml

```diff
@@ -25,10 +26,21 @@ gcsDeps:
     excludes: "/rc"
     readShaFromFile: true
     startIndex: 0
+    fusMetadataBucket: "layer-deps"
+    fusMetadataObject: "ubuntu1604/bazel/debs/metadata.yaml"
     releasePolicies:
       - tag: "default"
         # Everyday weekday at 5am.
         schedule: "0 0 5 * * Mon-Fri"
+      - tag: "cveMedium"
```

Comment:
lets add these release policies in an upcoming PR and only have this one to create the metadata file and see the FUS / DUS pass it around

Reply:
Done